src/generator/exercises/templates/example.tt

```diff
@@ -3,7 +3,7 @@ require "../src/*"
 
 describe <%= "#{describe_name.inspect} do" %>
 <% test_cases.each_with_index do |test_case, index| %>
-  <%= test_case.pending?(index) %> "tests <%= test_case.test_name %>" do
```

Comment:
At some point after this change we should probably refresh the other spec files for consistency (the make task in my PR for looping through all the generators should make refreshes nicer).

Also off topic but related, have you noticed that specs get generated with an extra newline on the first line inside the test block?